src/Tests.Microsoft.Security.Utilities.Core/WellKnownRegexPatternsTests.cs

```diff
@@ -25,6 +26,84 @@ public class WellKnownRegexPatternsTests
             "SEC101/109.AzureContainerRegistryLegacyKey"
         };
 
+        [TestMethod]
+        public void WellKnownRegexPatterns_EnsureAllPatternsExpressConfidence()
```

Comment:
What a fantastic test!

Reply:
:) This is how I find the things I needed to fix. I love this approach because:

1) you codify a development invariant that guarantees future contributors do not regress in an important concern
2) by authoring the test to drive resolving the gap, you guarantee that the test output itself is very actionable (to help those future devs who aren't aware of the standard).

rinse and repeat for all the quality bars you want to enforce in your plug-ins.